### PR TITLE
Cleaned up README.md formatting and fixed path for data-on-chain exam…

### PR DESCRIPTION
**Pull request by rakshasa**
…ple.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,10 +250,10 @@ The following process will help you deploy the [**`data_on_chain.py`**](https://
 
 This contract can be used in order to browse and get a feel of the COTI network. The contract allows for the secure handling of encrypted data, enabling storage, transformation, and arithmetic operations on encrypted values using the [**`MpcCore`**](https://github.com/coti-io/confidentiality-contracts/blob/main/contracts/lib/MpcCore.sol) library. It supports operations where values are encrypted using both network and user keys, ensuring data privacy and security on-chain.
 
-1.  Navigate to the `examples` directory in the `confidentiality-contracts` directory inside the `coti-sdk-python-examples` project.
+1.  Navigate to the `contracts` directory in the `confidentiality-contracts` directory inside the `coti-sdk-python-examples` project.
```
Comment (danielgruesso) [inline README.md:253]:
```suggestion
1.  Navigate to the `data_onchain` directory in the `confidentiality-contracts/examples` directory inside the `coti-sdk-python-examples` project.
```

File: README.md [hunk 2]
```diff
@@ -245,10 +250,10 @@ The following process will help you deploy the [**`data_on_chain.py`**](https://
...
 
     ```bash
-    cd confidentiality-contracts/contracts/examples
+    cd confidentiality-contracts/contracts
```
Comment (danielgruesso) [inline README.md:256]:
```suggestion
    cd coti-sdk-python-examples/examples/data_onchain
```

**Review by danielgruesso — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please review suggestions.

**Comment by rakshasa** [comment]
Those changes in the PR were made because it was required to execute the examples.

As such the suggestions are incorrect as far as the steps I did to successfully run the test.

E.g. adding `coti-sdk-python-examples` prefix ignores the fact that the user is already in that directory if they followed the instructions.

And when using `cd confidentiality-contracts/contracts/examples` it gives an error because the test is being run in the wrong directory.

**Comment by danielgruesso** [comment]
Thanks @rakshasa. Please pull the latest version, I believe some of the issues have already been addressed. Thanks again.

**Pull request closed without merging**